mixins.scss

```diff
@@ -884,11 +939,11 @@
 			$unit: unit($value);
 
 			@if $unit == 'px' {
-				$px-values: append($px-values, #{strip-unit($value)}px );
+				$px-values: append($px-values, #{strip-unit($value)} * 1px );
```

Comment:
I would change that in another PR, because i will concentrate here only on the documentation